daffodil-cli/src/it/scala/org/apache/daffodil/schematron/TestValidating.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.schematron
+
+import net.sf.expectit.matcher.Matchers.regexp
+import org.junit.Test
+
+class TestValidating {
+  val data = "input/uuid.txt"
+  val uuid = "xsd/string.dfdl.xsd"
+  val never = "sch/never-fails.sch"
+  val always = "sch/always-fails.sch"
+
+  val alwaysResult = regexp("<.+-fails>2f6481e6-542c-11eb-ae93-0242ac130002</.+-fails>\n")
```

Comment:
Tests in windows are failing, perhaps due to this \n? Is this \r\n on windows? Can we just make this tests loser so it doesn't care about the newline?

Reply:
Ill try that.  Looked like my paths were mangled in the logs, but maybe a red herring... I dont have a windows box to test so just going to take a few stabs at it and see what happens  :boom: :hammer: